code/AdobeEdge.brs

```diff
@@ -931,19 +932,39 @@ function _adb_buildEdgeRequestURL(configId as string, requestId as string, edgeD
 end function
 
 function _adb_EdgeRequestWorker(stateManager as object) as object
+    if stateManager = invalid
+        _adb_log_debug("stateManager is invalid")
+        return invalid
+    end if
+    ' if stateManager._type_ = invalid or stateManager._type_ <> "stateManager"
+    '     _adb_log_debug("not a StateManager object")
```

Comment:
Should we remove this? If this is an improvement let's open a task issue for this.

Reply:
Removed.